### ISSUE TITLE
Cannot create Redshift table in Apache Superset through sqlalchemy-redshift

### ISSUE THREAD

**Issue by larshelge** [sqlalchemy-redshift/sqlalchemy-redshift#203]
Hi, not sure if this is the right place to report this issue, let me know if I should report somewhere else.

## Description

I am trying to create a Redshift table in Apache Superset. I have first created the Redshift database, and verified that the connection is good. I have verified that the schema and the table I am creating actually exists in the Redshift cluster.

Seems related to #138.

## Expected behavior

I want the creation of the Redshift table to be successful.

## Versions

sqlalchemy-redshift: 0.7.8
sqlalcemy: 1.3.17
superset: 0.36.0
pip: 20.1.1
python: 3.6.9
os: Ubuntu 18.04

## Error logs

After creating the table in Superset, I get the following error in the logs:

```
Got an error in pre_add for public.dhis_dev
Traceback (most recent call last):
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/views.py", line 389, in pre_add
    table.get_sqla_table_object()
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/models.py", line 1074, in get_sqla_table_object
    return self.database.get_table(self.table_name, schema=self.schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/models/core.py", line 580, in get_table
    autoload_with=self.get_sqla_engine(),
  File "<string>", line 2, in __new__
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/deprecations.py", line 139, in warned
    return fn(*args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 559, in __new__
    metadata._remove_table(name, schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 554, in __new__
    table._init(name, metadata, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 648, in _init
    resolve_fks=resolve_fks,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 672, in _autoload
    _extend_on=_extend_on,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2221, in run_callable
    return conn.run_callable(callable_, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1660, in run_callable
    return callable_(self, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 467, in reflecttable
    table, include_columns, exclude_columns, resolve_fks, **opts
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 666, in reflecttable
    table_name, schema, **table.dialect_kwargs
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 392, in get_columns
    self.bind, table_name, schema, info_cache=self.info_cache, **kw
  File "<string>", line 2, in get_columns
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 52, in cache
    ret = fn(self, con, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 461, in get_columns
    comment=col.comment)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 664, in _get_column_info
    **kw
TypeError: _get_column_info() missing 1 required positional argument: 'generated'
ERROR:superset.connectors.sqla.views:Got an error in pre_add for public.dhis_dev
Traceback (most recent call last):
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/views.py", line 389, in pre_add
    table.get_sqla_table_object()
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/connectors/sqla/models.py", line 1074, in get_sqla_table_object
    return self.database.get_table(self.table_name, schema=self.schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/superset/models/core.py", line 580, in get_table
    autoload_with=self.get_sqla_engine(),
  File "<string>", line 2, in __new__
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/deprecations.py", line 139, in warned
    return fn(*args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 559, in __new__
    metadata._remove_table(name, schema)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/langhelpers.py", line 69, in __exit__
    exc_value, with_traceback=exc_tb,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/util/compat.py", line 178, in raise_
    raise exception
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 554, in __new__
    table._init(name, metadata, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 648, in _init
    resolve_fks=resolve_fks,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/sql/schema.py", line 672, in _autoload
    _extend_on=_extend_on,
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 2221, in run_callable
    return conn.run_callable(callable_, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/base.py", line 1660, in run_callable
    return callable_(self, *args, **kwargs)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/default.py", line 467, in reflecttable
    table, include_columns, exclude_columns, resolve_fks, **opts
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 666, in reflecttable
    table_name, schema, **table.dialect_kwargs
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 392, in get_columns
    self.bind, table_name, schema, info_cache=self.info_cache, **kw
  File "<string>", line 2, in get_columns
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy/engine/reflection.py", line 52, in cache
    ret = fn(self, con, *args, **kw)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 461, in get_columns
    comment=col.comment)
  File "/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/sqlalchemy_redshift/dialect.py", line 664, in _get_column_info
    **kw
TypeError: _get_column_info() missing 1 required positional argument: 'generated'
/home/lars/dev/lib/superset/venv/lib/python3.6/site-packages/flask_appbuilder/fields.py:182: UserWarning: allow_blank=True does not do anything for QuerySelectMultipleField.
  "allow_blank=True does not do anything for QuerySelectMultipleField."
```

**Comment by jklukas** [comment]
It looks like you're hitting the issue described in https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/195. Please try the newly released 0.7.9.

**Comment by larshelge** [comment]
Thanks for the reply @jklukas.

I upgraded to 0.7.9 and tested again. I can confirm the issue is now resolved

### Versions
sqlalchemy-redshift: 0.7.9
sqlalcemy: 1.3.17
python: 3.6.9
